great_expectations/rule_based_profiler/parameter_builder/parameter_builder.py

```diff
@@ -23,6 +24,14 @@
 from great_expectations.validator.metric_configuration import MetricConfiguration
 from great_expectations.validator.validator import Validator
 
+MetricComputationValues = Union[
+    Union[Any, Number, np.ndarray, List[Union[Any, Number]]]
+]
+MetricComputationDetails = Dict[str, Any]
+MetricComputationResult = make_dataclass(
+    "MetricComputationResult", ["metric_values", "details"]
+)
+
```

Comment:
These are placeholders until we can establish more robust types right? If so, please note that in a comment for reference

Reply:
@cdkini I added the comment (please see in that module).  Thanks!